Apps/phhackerone/hackerone_connector.py

```diff
@@ -182,28 +196,32 @@ def _delete_phantom_data(self, url):
             return None
 
     def _get_rest_data(self, url, url_params):
-        self.__print('Start: _get_rest_data(): {0}'.format(datetime.datetime.now()))
+        self.debug_print('Start: _get_rest_data(): {0}'.format(datetime.datetime.now()))
         try:
-            self.__print(url)
+            self.__print("Connecting to endpoint '{}'".format(url))
             u, p = self._get_auth()
             if url_params:
                 response = requests.get(url, auth=(u, p), params=url_params, headers=self._get_headers(), verify=False)
             else:
                 response = requests.get(url, auth=(u, p), headers=self._get_headers(), verify=False)
-            content = json.loads(response.text)
+            try:
+                content = json.loads(response.text)
+            except Exception as e:
+                self.__print("Error parsing JSON Object: {}".format(self._get_error_message_from_exception(e)))
+                return None, None
             code = response.status_code
             if code == 200:
                 if 'links' in content:
                     return content['data'], content['links']
                 else:
                     return content['data'], None
             else:
-                self.__print(code)
-                self.__print(content)
+                self.debug_print(code)
+                self.debug_print(content)
                 return None, None
         except Exception as e:
             err = self._get_error_message_from_exception(e)
-            print(err)
+            self.__print(err)
```

Comment:
I see that all of the self.__print calls were changed to self.debug_print, was this meant to be changed as well or no?

Reply:
No, this was not meant to be changed. We have only replaced self.__print with self.debug_print at the places where it is called by test_connectivity and on_poll, as it shows a very complex and long message in the output because of self.save_progress.